gsw_oceanographic_toolbox.c

```diff
@@ -1,6 +1,6 @@
 /*
 **  $Id: gsw_oceanographic_toolbox-head,v c61271a7810d 2016/08/19 20:04:03 fdelahoyde $
-**  Version: 3.05.0-4
+**  Version: 4
```

Comment:
One of the things we need to do is decide on, and document, a version strategy for each of the languages.  There will inevitably be divergence among them, though, because I don't think any of the others will ever actually match the Matlab; and even between R and Python, I expect there will be increasing differences.